pkg/transform/transform.go

```diff
@@ -63,6 +72,54 @@ func ToOPAInputValue(x any) (ast.Value, error) {
 	return value, nil
 }
 
+// ToAST converts a Rego module to an ast.Value suitable for use as input in Regal
```

Comment:
```suggestion
// ToAST converts a Rego module to an ast.Value suitable for use as input in Regal.
```


I thought this would be a linter error? 🤔